Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.6.3
+
+COPY . /go/src/github.com/cloudflare/redoctober
+RUN go get github.com/cloudflare/redoctober/...
+
+EXPOSE 8080
+ENTRYPOINT ["redoctober"]
```

Comment:
Red October will want a TLS cert/key, which this won't provide. Perhaps including a small script for generating a default set would be useful?